### PR TITLE
CI: check the build with OCaml 4.07.1 and 4.08.0

### PR DESCRIPTION
**Pull request by Julow**
Fix https://github.com/ocaml-ppx/ocamlformat/issues/1035

Small tweak to build with 4.07.

### CODE REVIEW

**Comment by gpetiot** [comment]
It looks good to me, if you approve my travis changes you can merge.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
-  - opam switch ${OCAML_VERSION}
-  - opam update --upgrade
-  - opam pin add --no-action ocamlformat .
-  - opam install --deps-only ocamlformat
+  - |
+    case "$TO_TEST" in
+      install|tests)
+        opam switch ${OCAML_VERSION}
+        opam update --upgrade
+        opam pin add --no-action ocamlformat .
```
Comment (Julow) [inline .travis.yml]: We should do that below, in `script`. Otherwise it may be cached.

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
+      install|tests)
+        wget -O ${HOME}/opam https://github.com/ocaml/opam/releases/download/2.0.2/opam-2.0.2-x86_64-linux
+        chmod +x ${HOME}/opam
+        export PATH=${HOME}:${PATH}
+        export OPAMYES=1
+        export OPAMJOBS=2
+        opam init --bare --disable-sandboxing
+        opam switch --dry-run ${OCAML_VERSION} 2>/dev/null || opam switch create ${OCAML_VERSION}
+        ;;
+    esac
```
Comment (Julow) [inline .travis.yml]: You should add a case for the empty string, same in `install` and `script`.
If `TO_TEST` is not set correctly, nothing will happen, that should be an error.

Comment (gpetiot) [reply .travis.yml]: done

File: .travis.yml (hunk 1 above)
Comment (gpetiot) [reply .travis.yml]: done

**Pull request merged**